FBReader/src/main/java/org/geometerplus/android/fbreader/library/DownloadedBookInfoActivity.java

```diff
@@ -109,11 +110,17 @@ protected void onCreate(Bundle icicle) {
 	}
 
 	@Override
-	protected void onStart() {
-		super.onStart();
+	protected boolean shouldShowAddToReadingListButton(){
+		if(isBookshareIdAvailable) return super.shouldShowAddToReadingListButton();
+		else return false;
```

Comment:
Here, you can remove the "else" and just return false.  No need to change it now, just for future reference.